packages/flutter_tools/lib/src/windows/windows_version_validator.dart

```diff
@@ -50,7 +50,7 @@ class WindowsVersionValidator extends DoctorValidator {
     // semantic versioned strings
     // (ie. 10.5.4123)
     final RegExp regex =
-        RegExp(r'\n(OS Version:\s*)([0-9]+\.[0-9]+\.[0-9]+)(.*)\n');
+        RegExp(r'\n?(OS Version:\s*)([0-9]+\.[0-9]+\.[0-9]+)(.*)\n?');
```

Comment:
Does adding the question mark not make this also match `'BIOS Version: `?

Reply:
I don't believe so, the question mark should be only looking for 0 or 1 newline characters before `OS`. Unless I had the wrong assumption there?

I added the question mark incase there was no new line character before the `OS Version` line on print out

Reply:
So in "BIOS Version:" won't there be 0 newline characters before "OS"?

Reply:
whether i'm right or wrong, this should be test cases in the unit tests :)

Reply:
I see what you mean now, I'm going to try a refactor with `multiLine` enabled so that I can specify the start of the line as a reference point

Reply:
I'll build some test cases with multiline enabled when i get back. thanks for the review! 😁